package.json

```diff
@@ -332,6 +333,7 @@
         "@box/react-virtualized": "9.22.3-rc-box.2",
         "@hapi/address": ">=2.0.0 <2.1.2",
         "axios": "^0.25.0",
+        "box-annotations": "^4.1.1-beta.1",
```

Comment:
Can we clarify how we intend to handle this dependency in the future? Will this package eventually be moved to [box-content-preview](https://github.com/box/box-content-preview/blob/master/package.json#L38) to replace the existing `box-annotations` dependency there? Do we intend to continue supporting two versions of annotations?

Reply:
removed the dependency after speaking with product